src/blocks.tsx

```diff
@@ -182,8 +182,15 @@ export const InformationBlock: React.FC<InformationBlockProps> = ({
 interface HighlightedValueBlockProps {
     tag?: RendersNullableElement<typeof Tag>;
 
-    value: string | Array<string>;
-    text: string | Array<string>;
+    mainHeading: {
+        value: string;
+        text: string;
+    };
+
+    secondHeading?: {
+        value: string;
+        text: string;
+    };
 
     secondaryValue?: string;
```

Comment:
is this used?

Reply:
yes, it is used in this part of the code:

<img width="636" alt="Captura de tela 2023-06-29 092150" src="https://github.com/Telefonica/mistica-web/assets/123959276/c2d2f0a9-27f4-4ad2-8b0c-0955009a0594">